app/assets/javascripts/select-kit/components/composer-actions.js.es6

```diff
@@ -192,6 +192,17 @@ export default DropdownSelectBoxComponent.extend({
       });
     }
 
+    const currentUser = Discourse.User.current();
```

Comment:
@eviltrout I can't recall where the discussion happened in the past but do you remember why we wanted to avoid accessing models via the `Discourse` global?